src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
+
+  /** Max ring radius chosen such that it has the same area as the total geometry */
```

Comment:
Can this be something like:

    /** Max ring radius chosen such that it has the same area as the bounding box of the universe it lies in */

We can solve more than just pin cells with OpenMOC!